examples/python-telegram-bot/README.md

```diff
@@ -20,9 +20,9 @@ This example starts a Telegram bot using [python-telegram-bot](https://www.pytho
 
 ## 💁‍♀️ How to use
 
-- Install packages using `pip install -r requirements.txt`
-- Start the bot using `python bot.py`
+- Install packages using `pip3 install -r requirements.txt`
```

Comment:
These 2 changes really aren't needed. It runs with Python 3 anyway.

Reply:
😅